src/Message/AuthorizeRequest.php

```diff
@@ -7,6 +7,21 @@
  */
 class AuthorizeRequest extends AbstractRequest
 {
+    public function getApplicationFee()
+    {
+        return $this->getParameter('applicationFee');
+    }
+
+    public function getApplicationFeeInteger()
+    {
+        return (int) round($this->getApplicationFee() * pow(10, $this->getCurrencyDecimalPlaces()));
```

Comment:
As an application fee is effectively a secondary charge amount I just mirrored omnipay's existing amount methods but I'm happy to remove ``getApplicationFeeInteger()`` and move the calculation into ``getApplicationFee()`` as Stripe always requires the value as an integer.